proposed/project-file-simplification.md

```diff
@@ -0,0 +1,351 @@
+
+We are exploring broader use of the `AssignOnce` attribute on properties that make them immutable. We think this may solve the request to block further property changes in `Import` files.
+
+#### Abbreviating version declaration
+
+The SDK version number changes quarterly, while most breaking changes and warnings are introduced annually. Also, when users think about the version their behavior is tied to, users may think in terms of aligning to the major .NET version. Thus, supporting a short hand of just `10`, rather than `10.0.100` is desirable. If the user wants to use a quarterly `Base` file, they will need to enter the full SDK version number. We anticipate quarterly changes to `Base` files to be rare, except in support of upcoming preview features.
+
+### Changes to common user files
+
+We will add two new attributes to project files. This will be an optional/alternative way to declare the SDK and the property values that were previously in the project file when created from templates. The existing approach to project files will remain unchanged.
+
+In addition, we will provide a mechanism for declaring a default property value. When this file is read a check will be made - if the property has already been set, the existing value will be retained and no action will be performed.
+
+#### `Type` attribute
+
+_The name `Type` is not final._
+
+The `SDK` attribute on the `Project` element can be replaced by a `Type` attribute. It is not legal to have both a `Type` and an `SDK` attribute. If neither exists, the project will behave in the same manner as projects without a `SDK` attribute behave today.
+
+From a user perspective, the `Type` attribute is a friendlier alternative to the `SDK` attribute since its name and contents describe intent. Also, this allows application type specific properties to be moved from the project file into the `Type` system.
+
+It is an error for a `Type` attribute to appear anywhere but a project file `Project` element. For example, it cannot appear in `Directory.Build.props`, The reason for this restriction is that the logic run when the `Type` attribute appears determines the SDK (the tasks and targets) that are used. `Directory.Build.props` is located as part of an SDK task.
+
+There is a file delivered as part of the SDK that provides the `Type` information. The effect of this file will be the same as if it was a props file with an `SDK` attribute on the `Project` element and a series of properties. It is an error for a `Type` file to be missing the SDK, to contain any MSBuild element except `PropertyGroup`. Since we will create and deliver these files, we might not do the work to enforce this restriction. The purpose of this restriction is that SDKs already allow adding information to the build, and anything more complex is probably better done via an SDK.
```

Comment:
```suggestion
There is a file delivered as part of the SDK that provides the `Type` information. The effect of this file will be the same as if it was a `.props` file with an `SDK` attribute on the `Project` element and a series of properties. It is an error for a `Type` file to be missing the SDK or to contain any MSBuild element except `PropertyGroup`. Since we will create and deliver these files, we might not do the work to enforce this restriction. The purpose of this restriction is that SDKs already allow adding information to the build, and anything more complex is probably better done via an SDK.
```